hip-config-clang.cmake.in

```diff
@@ -50,14 +50,24 @@ set_and_check( hip_BIN_INSTALL_DIR "@PACKAGE_BIN_INSTALL_DIR@" )
 set_and_check(hip_HIPCC_EXECUTABLE "${hip_BIN_INSTALL_DIR}/hipcc")
 set_and_check(hip_HIPCONFIG_EXECUTABLE "${hip_BIN_INSTALL_DIR}/hipconfig")
 
-get_filename_component(HIP_CLANG_ROOT "${CMAKE_CXX_COMPILER}" PATH)
-get_filename_component(HIP_CLANG_ROOT "${HIP_CLANG_ROOT}" PATH)
+if(CMAKE_CXX_COMPILER MATCHES ".*hipcc")
+  execute_process(COMMAND ${CMAKE_CXX_COMPILER} --version 
+                  OUTPUT_STRIP_TRAILING_WHITESPACE
+                  OUTPUT_VARIABLE HIP_CLANG_CXX_COMPILER_VERSION_OUTPUT)
+  if(HIP_CLANG_CXX_COMPILER_VERSION_OUTPUT MATCHER "InstalledDir: ([^\t\r\n])")
```

Comment:
I tried this and got error:
/opt/rocm/hip/lib/cmake/hip/hip-config.cmake(78):  execute_process(COMMAND ${CMAKE_CXX_COMPILER} --version OUTPUT_STRIP_TRAILING_WHITESPACE OUTPUT_VARIABLE HIP_CLANG_CXX_COMPILER_VERSION_OUTPUT )
/opt/rocm/hip/lib/cmake/hip/hip-config.cmake(81):  if(HIP_CLANG_CXX_COMPILER_VERSION_OUTPUT MATCHER InstalledDir: ([^\t\r\n]) )
CMake Error at /opt/rocm/hip/lib/cmake/hip/hip-config.cmake:81 (if):
  if given arguments:

    "HIP_CLANG_CXX_COMPILER_VERSION_OUTPUT" "MATCHER" "InstalledDir: ([^	

  ])"

  Unknown arguments specified
Call Stack (most recent call first):
  cmake/VerifyCompiler.cmake:58 (find_package)
  CMakeLists.txt:47 (include)


-- Configuring incomplete, errors occurred!

Reply:
there is a typo:
MATCHER -> MATCHES

Reply:
Oops that should be `MATCHES`